Change time_limit_min to account for end_date (#2405)
PrairieLearn/PrairieLearn: docs/remoteExams.md

Some notes about this configuration:

> * The exam window (70 minutes, `startDate` to `endDate`) has been set to be 10 minutes longer than the exam time limit (60 minutes). However, they will not be able to access the exam past the `endDate` time under any circumstances. If a student starts this exam more than 10 minutes late, then the countdown timer on their exam will reflect the time remaining until `endDate` (previously, the timer would incorrectly show the full exam time limit regardless of `endDate`).

Reviewer: ```suggestion
* The exam window (70 minutes, `startDate` to `endDate`) has been set to be 10 minutes longer than the exam time limit (60 minutes). However, students will not be able to access the exam past the `endDate` time under any circumstances. If a student starts this exam more than 10 minutes late, then the countdown timer on their exam will reflect the time remaining until `endDate`.
```

I think it's best to just describe how things currently work. Talking about how things used to be is helpful in the short run, but it quickly gets confusing over time.

> * The exam window (70 minutes, `startDate` to `endDate`) has been set to be 10 minutes longer than the exam time limit (60 minutes). However, they will not be able to access the exam past the `endDate` time under any circumstances. If a student starts this exam more than 10 minutes late, then the countdown timer on their exam will reflect the time remaining until `endDate` (previously, the timer would incorrectly show the full exam time limit regardless of `endDate`).

Reviewer: Now that we've fixed this endDate problem, I actually think we should shorten the "extra time" from 10 minutes to 5 minutes.

> * The exam window (70 minutes, `startDate` to `endDate`) has been set to be 10 minutes longer than the exam time limit (60 minutes). However, they will not be able to access the exam past the `endDate` time under any circumstances. If a student starts this exam more than 10 minutes late, then the countdown timer on their exam will reflect the time remaining until `endDate` (previously, the timer would incorrectly show the full exam time limit regardless of `endDate`).

Reviewer: I guess we should also document the `endDate` behavior of time limits at https://prairielearn.readthedocs.io/en/latest/accessControl/#time-limits
